kimchi/src/tests/framework.rs

```diff
@@ -122,7 +122,6 @@ where
     }
 
     /// creates the indexes
-    #[must_use]
```

Comment:
Offtop but: I removed this since it was giving a warning: `setup_with_custom_srs` is indeed used, but only in tests. `must_use` does not understand this and produces a warning.

Reply:
This did not work.